pom.xml

```diff
@@ -205,6 +205,14 @@
 
     <dependencyManagement>
         <dependencies>
+            <!--opentelemetry-->
+            <dependency>
+                <groupId>io.opentelemetry</groupId>
+                <artifactId>opentelemetry-bom</artifactId>
+                <version>1.24.0</version>
```

Comment:
Please define the version as a reference

Reply:
The parameter name is consistent with the rewritten function.